src/ffi/message.rs

```diff
@@ -109,17 +122,18 @@ pub enum Payload {
         bytes: Vec<u8>,
     },
 }
-#[uniffi::export]
+// TODO: This passes cargo build but does not work with uniffi-bindgen.
+//#[uniffi::export]
 impl Payload {
     /// Deserializes JSON serialized to a string to [Payload::JSON] or errors with a
     /// [JSONDeserializationError] if the string is invalid JSON.
-    #[uniffi::constructor]
+    //#[uniffi::constructor]
     pub fn json_from_serialized(serialized_json: String) -> Result<Self, JSONDeserializationError> {
         JSON::deserialize(serialized_json).map(|json| Self::JSON { json })
     }
 
     /// Stores bytes in [Payload::Binary].
-    #[uniffi::constructor]
+    //#[uniffi::constructor]
```

Comment:
Like `JSON` this is code that doesn't include my current understanding that only `uniffi::Object` can have `uniffi::constructor`. Since `Payload` is a `uniffi::Enum`, this `uniffi::constructor` should be removed.